x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form/aws_input_var_fields.tsx

```diff
@@ -13,7 +13,13 @@ export const AwsInputVarFields = ({
   fields,
   onChange,
 }: {
-  fields: Array<AwsOptions[keyof AwsOptions]['fields'][number] & { value: string; id: string }>;
+  fields: Array<
+    AwsOptions[keyof AwsOptions]['fields'][number] & {
+      value: string;
+      id: string;
+      dataTestId: string;
```

Comment:
we have a lot of inconsistency in the way we name this type of parameter, to which I also contributed. We already have `testId`, `testSubj`, `dataTestSubj`. I wouldn't introduce the fourth variant `dataTestId`, instead let's stick to one and make sure we use it in this PR. And in the follow up PR rename all other places to this one name. I'd propose `dataTestSubj` to match the prop name `data-test-subj` used on EUI components